src/plots/gl3d/layout/defaults.js

```diff
@@ -110,6 +110,31 @@ function handleGl3dDefaults(sceneLayoutIn, sceneLayoutOut, coerce, opts) {
         sceneLayoutIn, sceneLayoutOut, opts
     );
 
-    coerce('dragmode', opts.getDfltFromLayout('dragmode'));
+    var dragmode = opts.getDfltFromLayout('dragmode');
+
+    if(dragmode !== false) {
+        if(!dragmode) {
+
+            dragmode = 'orbit';
+
+            if(sceneLayoutIn.camera &&
+                sceneLayoutIn.camera.up) {
+
+                var x = sceneLayoutIn.camera.up.x;
+                var y = sceneLayoutIn.camera.up.y;
+                var z = sceneLayoutIn.camera.up.z;
+
+                if(!x || !y || !z) {
+                    dragmode = 'turntable';
+                } else if(z / Math.sqrt(x * x + y * y + z * z) > 0.999) {
```

Comment:
This PR is getting better by the minute. Thanks for your efforts @archmoj 

Now, would there be a way to tweak this condition such that `gl3d_ibm-plot` and `gl3d_surface-lighting` still default to `dragmode: 'turntable'`?

Reply:
Good point @etpinard. Let me check.